### ISSUE TITLE
Navigate to class does not support pod layout

### ISSUE THREAD

**Issue by jdenly** [Turbo87/intellij-emberjs#15]
When using pod layout, the navigate to class feature does not work.

For example
```
/routes/something/test.js
```
Correctly resolves when navigating to classes.
```
/something/test/route.js
```
Shows as a route (as indicated by the icon changing), but does not show when trying to navigate to class.

**Comment by Turbo87** [comment]
hmm, as you can see by the changing icon the file is correctly recognized as a route so it should show up in the goto class dialog... I'll see if I can reproduce this. 

Just to clarify, you don't expect this to be a route unit test, right?

**Comment by jdenly** [comment]
Nope, I was expecting it should show up as a route.

**Comment by jdenly** [comment]
How strange, I just tried to recreate this with a fresh project and it works fine.

I ran the following steps, with these versions.
```
ember -v
version: 1.13.8
node: 0.12.6
npm: 2.13.4
os: darwin x64
```

```
mkdir test-project
cd test-project
ember init
ember generate route NormalRoute
ember generate route PodRoute --pod
```
I then opened in IDEA 15.0.1, waited for indexing to finish and both routes have the correct icon. Using navigate to class now works for both. It must be a peculiarity of my other project.

I'll see what is different between the two projects and get back to you.
